Subject: Don't need to downgrade optimisation on ppc/ia64 with gcc 4.x


---
 hints/linux.sh |   17 -----------------
 1 files changed, 0 insertions(+), 17 deletions(-)

diff --git a/hints/linux.sh b/hints/linux.sh
index c88f157..01ba1a1 100644
--- a/hints/linux.sh
+++ b/hints/linux.sh
@@ -126,23 +126,6 @@ case "$optimize" in
 # use -O2 by default ; -O3 doesn't seem to bring significant benefits with gcc
 '')
     optimize='-O2'
-    case "`uname -m`" in
-        ppc*)
-            # on ppc, it seems that gcc (at least gcc 3.3.2) isn't happy
-            # with -O2 ; so downgrade to -O1.
-            optimize='-O1'
-        ;;
-        ia64*)
-            # This architecture has had various problems with gcc's
-            # in the 3.2, 3.3, and 3.4 releases when optimized to -O2.  See
-            # RT #37156 for a discussion of the problem.
-            case "`${cc:-gcc} -v 2>&1`" in
-            *"version 3.2"*|*"version 3.3"*|*"version 3.4"*)
-                ccflags="-fno-delete-null-pointer-checks $ccflags"
-            ;;
-            esac
-        ;;
-    esac
     ;;
 esac
 
-- 
tg: (bc65f46..) debian/ppc_ia64_optim (depends on: upstream)
